e2e/patient/patient-detail/patient-dialog.po.ts

```diff
@@ -48,4 +53,20 @@ export class PatientDetailPage extends BasePage {
 	public getButtonSubmit() {
 		return this.getObjectById('PatientSubmitButton');
 	}
+
+	public getAllTabs() {
```

Comment:
The method getAllTabs() is generic and must be in the BasePage just once.

Reply:
done